js/script.js

```diff
@@ -1,29 +1,65 @@
-function calcSum() { 
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c1 = a + b;
-    document.getElementById('result').innerHTML = 'result ' + c1;   
- } 
- 
- function calcSub() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c2 = a - b;
-    document.getElementById('result').innerHTML = 'result ' + c2;   
- }
+let globalNumbers = document.querySelectorAll('.action-bar__numbers'),
+    operators = document.querySelectorAll('.action-bar__operators'),
```

Comment:
Use separate declarations for each variable: 

```
const globalNumbers = ...;
const operators = ...;
let operatorClicked = ...;
etc.
```

It will improve code readability and maintenance.